site/assets/scss/_navbar.scss

```diff
@@ -1,57 +1,45 @@
 .bd-navbar {
   // Boosted mod: no padding
   // Boosted mod: no background-color
-  // Boosted mod: no background-image
   // Boosted mod: no box-shadow
 
+  // Boosted mod: no `&::after`
+
   // Boosted mod: no .bd-navbar-toggle
 
   // Boosted mod: no .navbar-toggler
-  .nav-link { // Boosted mod: Needed since the dropdowns define their own color (could be removed once they are tackled)
-    // Boosted mod: no display, no padding, no color
 
+  // Boosted mod: no .navbar-brand
+
+  // Boosted mod: Needed since the dropdowns define their own color (could be removed once they are tackled)
+  .nav-link {
     &:hover {
-      color: var(--#{$prefix}link-hover-color); // Boosted mod
+      color: var(--#{$prefix}link-hover-color);
     }
 
     &.active {
-      // Boosted mod: no font-weight
-      color: var(--#{$prefix}link-hover-color); // Boosted mod
+      color: var(--#{$prefix}link-hover-color);
     }
   }
+  // End mod
 
   // Boosted mod: no .navbar-nav-svg
 
   // Boosted mod: no .offcanvas-lg
 
   // Boosted mod: no .dropdown-toggle
 
+  // Boosted mod
   .dropdown-menu {
     --#{$prefix}dropdown-min-width: 12rem;
-    // Boosted mod: background-color is already good
-    @include rfs($font-size-sm, --#{$prefix}dropdown-font-size); // Boosted mod
-    --#{$prefix}dropdown-line-height: #{$line-height-sm}; // Boosted mod
-    --#{$prefix}dropdown-item-padding-y: #{$dropdown-item-padding-y + .0625rem}; // Boosted mod
-    // Boosted mod: no box-shadow
-  }
-
-  // Boosted mod: Use mask instead of background + ::after
-  .dropdown-item.current {
-    position: relative;
-
-    &::after {
-      position: absolute;
-      top: .75rem;
-      right: $dropdown-item-padding-x;
-      width: .75rem;
-      height: .75rem;
-      content: "";
-      background-color: currentcolor;
-      mask: escape-svg($dropdown-active-icon) no-repeat 50% / .75rem;
-    }
+    @include rfs($font-size-sm, --#{$prefix}dropdown-font-size);
+    --#{$prefix}dropdown-line-height: #{$line-height-sm};
+    --#{$prefix}dropdown-item-padding-y: #{$dropdown-item-padding-y + .0625rem};
```

Comment:
Use `--bs-` instead of `--#{$prefix}` ? Or will it be fixed via another PR after ?

Reply:
Yes + check the entire file

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/c5d4b026b32aea26e87f2b7332632a613f929b00